### DISCUSSION TITLE
Awaiting benefits

### DISCUSSION THREAD

**Discussion by EllieMoynihan** [community/community#62949] [GitHub Education]
### Hi everyone,

Congrats! Your request for @EllieMoynihan was approved and your benefits will be available soon

It's been 4 day i haven't heard anything from them 
0 benefits i got i need for programming stuff

**Comment by GhostOf0days** [comment]
Please give it 3 more days (benefits should appear at a maximum of 5-7 days after approval). If it hasn't given you the benefits by then, please contact [Github Support](https://support.github.com/).
